extensions/agroal/deployment/src/test/java/io/quarkus/agroal/test/GcpDataSourceConfigTest.java

```diff
@@ -0,0 +1,69 @@
+package io.quarkus.agroal.test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.sql.SQLException;
+import java.time.Duration;
+
+import javax.inject.Inject;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.agroal.api.AgroalDataSource;
+import io.agroal.api.configuration.AgroalConnectionFactoryConfiguration;
+import io.agroal.api.configuration.AgroalConnectionPoolConfiguration;
+import io.agroal.narayana.NarayanaTransactionIntegration;
+import io.quarkus.test.QuarkusUnitTest;
+
+public class GcpDataSourceConfigTest {
```

Comment:
Not sure what this test is supposed to do. It doesn't seem to test anything for Google Cloud Platform than checking if the values in the properties file were properly applied.

Reply:
@gastaldi You are correct, the test was not specific to GCP so I removed that test and updated the `DefaultDataSourceConfigTest` with the extra assertions.

Reply:
Excellent! Thanks